handlers/playmobile/playmobile.go

```diff
@@ -117,6 +118,21 @@ func (h *handler) receiveMessage(ctx context.Context, c courier.Channel, w http.
 			return nil, handlers.WriteAndLogRequestError(ctx, h, c, w, r, err)
 		}
 
+		// remove message prefix according to a list of possible prefixes, useful for free accounts
+		incomingPrefixes := c.ConfigForKey(configIncomingPrefixes, []string{})
+		if prefixes, ok := incomingPrefixes.([]string); ok {
+			for _, prefix := range prefixes {
+				text := pmMsg.Content.Text
+				prefix = strings.ToLower(prefix)
+
+				if strings.ToLower(text[:len(prefix)]) == prefix {
```

Comment:
`strings.HasPrefix` and `strings.TrimPrefix` are here for you! :)

Reply:
Ya, but we need a case-insensitive check (with space or not after the prefix).

Reply:
`strings.HasPrefix(strings.ToLower(text), prefix)`?

Note that your current implementation will blow up for cases where text is shorter than prefix which is one argument to avoid slices where you can.

Reply:
Hmm, I just preferred to convert into lowercase the possible prefix instead of all the text... really, I disregarded the case that the `incoming_prefixes` config is set and the text doesn't have a prefix, I gonna fix it.